source/lib/index.ts

```diff
@@ -49,30 +58,31 @@ const findTestFiles = async (typingsFile: string, options: Options & {config: Co
  * @returns A promise which resolves the diagnostics of the type definition.
  */
 export default async (options: Options = {cwd: process.cwd()}) => {
+	let typingsFile = options.typingsFile ? options.typingsFile : '';
 	const pkgResult = await readPkgUp({cwd: options.cwd});
 
 	if (!pkgResult) {
 		throw new Error('No `package.json` file found. Make sure you are running the command in a Node.js project.');
 	}
 
 	const pkg = pkgResult.packageJson;
-
 	const config = loadConfig(pkg as any, options.cwd);
 
-	// Look for a typings file, otherwise use `index.d.ts` in the root directory. If the file is not found, throw an error.
-	const typingsFile = await findTypingsFile(pkg, options);
+	// Look for a typings file if not explicitly specified, otherwise use `index.d.ts` in the root directory. 
+	// If the file is not found, throw an error.
+	typingsFile = await findTypingsFile(pkg, options);
 
 	const testFiles = await findTestFiles(typingsFile, {
 		...options,
 		config
 	});
 
 	const context: Context = {
-		cwd: options.cwd,
 		pkg,
-		typingsFile,
+		config,
 		testFiles,
-		config
+		typingsFile,
+		cwd: options.cwd
 	};
```

Comment:
Type check?

Reply:
Isn't that what `Context` interface automatically infers? @kunal-kushwaha

Reply:
Ah I misunderstood something, yes you're right 👍